coalib/parsing/ConfParser.py

```diff
@@ -7,20 +7,22 @@
 from coalib.parsing.LineParser import LineParser
 from coalib.settings.Section import Section
 from coalib.settings.Setting import Setting
+from coalib.bearlib import deprecate_settings
```

Comment:
`deprecate_settings` is supposed to be used only with bears.  So it is being 'misused' here, but ... this is innovation .. if it works, that is great.

Could you add a test for `ConfParser` that uses the old name `comment_seperators=(';')` , and traps the logged warning that `deprecate_settings` emits.

The warning will say "The setting .." which is not 100% correct as these are not settings.  However if it works correctly, we'll ignore that problem as it is close enough, and we'll enhance `comment_seperators` in another issue & PR.